src/contracts/acrossConfigStore.test.ts

```diff
@@ -1,4 +1,5 @@
 import { Client } from "./acrossConfigStore";
+jest.useFakeTimers();
```

Comment:
These fix some warnings described by this issue: https://stackoverflow.com/questions/50793885/referenceerror-you-are-trying-to-import-a-file-after-the-jest-environment-has